Paymetheus/CreateAccountDialogView.xaml

```diff
@@ -14,12 +14,12 @@
                 <Button Content="Cancel" IsCancel="True" Command="{Binding HideDialogCommand}" Style="{DynamicResource ButtonWhite}" Width="100" Margin="5"/>
             </StackPanel>
             <StackPanel>
-                <Label Content="Create account" FontSize="18"/>
-                <Label Content="Account name:"/>
+                <Label Content="Create account" FontSize="18" Foreground="#FF0C1E3E"/>
```

Comment:
OK; I can revert but the styles for the dialogue views are currently a disaster. They should be using the dark font colour rather than black, amongst other issues.